### ISSUE TITLE
[FR] Request for Linux vst version of all your plugins.

### ISSUE THREAD

**Issue by mikobuntu** [sjaehn/BOops#2]
As always, I think I have been pressing you to think about porting your plugins to VST format [ most likely VST3 with License issues etc now ] . 

I think you could be rewarded financially by doing so. 

As you already know most native Linux DAWS don't/won't support LV2 and bridging adds some CPU overhead.

I believe you could really make the difference to people converting to Linux Audio as a serious and better OS for creating Audio media.

An example of your work so far... I waited for years until Illformed released a native Linux version of their Glitch plugin, then you basically release BOops overnight :+1: 

Anyway , please keep this issue open for other Linux musicians to comment and thanks for all your hard work!

**Comment by magnetophon** [comment]
> most native Linux DAWS don't/won't support LV2 

Which ones?
All the linux DAW's I know support LV2.

**Comment by sub26nico** [comment]
Only closed-source DAWs doesn't support lv2 on GNU/Linux systems... All FLOSS Daws support it.

**Comment by mikobuntu** [comment]
Correct, I should have mention that I was referring to proprietary DAWs such as Reaper, Bitwig, Renoise and Waveform, which I use more now compared to LMMS ( LV2 is being worked on by developers ) and Ardour.

**Comment by sjaehn** [comment]
VST is not my primary target. But also not out of scope. I'll take a look about the workload to port the B.Music plugins to Karl Steinbergs plugin format next year. But I can't guarantee if a VST will come or not.

**Comment by ghost** [comment]
> Only closed-source DAWs doesn't support lv2 on GNU/Linux systems... All FLOSS Daws support it.

btw radium is an open-source daw, and it doesn't support lv2

**Comment by Amplifi-Beats** [comment]
It'd be great if this plugin was ported to windows as well under the VST (2/3) format, you'd reach a huge audience, who would be very happy to get this plugin

**Comment by sjaehn** [comment]
Windows VSTs are two or three steps away. I'll first look for LV2s for all common platforms. I'll start after release of the version 1.8.0. But I can't promise if I'll succeed or not.

**Comment by trevormeier** [comment]
One possibility is to consider the new CLAP open plugin format (being backed by Bitwig, u-he et al). Its aim is for a technically complete, open and cross-platform plugin format that’s simple to develop for, has broad support from DAWs and plugin developers, and can export to all of the major other plugin formats. Worth a look as it appears it may simplify the non-DSP workload of developing a plugin.
